doc/internal/man3/ossl_algorithm_do_all.pod

```diff
@@ -41,11 +41,7 @@ The function described here are mainly useful for discovery, and
 possibly display of what has been discovered, for example an
 application that wants to display the loaded providers and what they
 may offer, but also for constructors, such as
-L<ossl_construct_method(3)>.
-
-=head1 SEE ALSO
-
-L<ossl_construct_method(3)>, L<EVP_MAC_do_all(3)>
+ossl_construct_method().
```

Comment:
Why did you remove the link surrounding `ossl_construct_method`? ....  Ohhhh, it's misspelled!  it should be `ossl_method_construct`!

Also, `EVP_MAC_do_all` should be `EVP_MAC_do_all_ex`

Reply:
thanks!  fixed.